types/amap-js-api/test/arryBounds.ts

```diff
@@ -0,0 +1,18 @@
+import {
+    lnglat
+} from './preset';
+
+// $ExpectType ArrayBounds
+const arrayBounds = new AMap.ArrayBounds([lnglat, lnglat, lnglat]);
```

Comment:
I also don't understand why the test shows the use of the global `AMap` inside a module. Is that what actual usage looks like? I would expect it to be used in a non-module file since it is a global.